Makefile.am

```diff
@@ -2598,7 +2598,7 @@ ganeti:
 .PHONY: check-dirs
 check-dirs: $(GENERATED_FILES)
 	@set -e; \
-	find . -type d \( -name . -o -name .git -prune -o -print \) | { \
+	find . -type d -not -name '.' -not -path '*/\.*' | { \
```

Comment:
@rbott, why is this change needed? The reason .git is ignored is because the build system knows how to filter it out from the final sources. If there's anything the build system does _not_ know how to handle, then I'd like it to fail in `check-local`.

Reply:
if you run `LC_ALL=C make check-local` it prints out:
```
Directory ./.github not listed in Makefile
Directory ./.github/workflows not listed in Makefile
make: *** [Makefile:4843: check-dirs] Error 1
```
The intention I assume, was to exclude `.dot-folders` at all (including .github). @apoikos: it sounds like this is the wrong way?

Reply:
I have updated this PR to only exclude .git and .github folders. This way we do not falsly exclude needed folders and also have a clean list of what is supposed to be excluded.